### DISCUSSION TITLE
will you guys support linux (at least ubuntu?)

### DISCUSSION THREAD

**Discussion by sergenti** [illixion/vscode-vibrancy-continued#69] [General]
love your work, looks dope

**Comment by illixion** [comment]
This is currently tracked in #27, from some of the research I've done the current issue seems to be that it's difficult to implement Electron transparency in Linux, and especially in a way that works across different hardware and distros. I won't be working on this personally as I mostly just maintain the extension, but if anyone makes a PR that adds Linux support, I'll definitely merge it and update that issue.
